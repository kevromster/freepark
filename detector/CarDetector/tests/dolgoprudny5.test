config = camera_msk_dolgoprudny_nb5.txt
input = input/dolgoprudny5.png
expected = expected/dolgoprudny5.out
