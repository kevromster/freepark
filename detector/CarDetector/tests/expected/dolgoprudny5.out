[
parking id: 4
spots count: 4

{ lat: 55.9374; lon: 37.506072; complexity: EXPERIENCED }
{ lat: 55.937397; lon: 37.506149; complexity: NEWBIE }
{ lat: 55.937396; lon: 37.506185; complexity: NEWBIE }
{ lat: 55.937392; lon: 37.506286; complexity: NEWBIE }
]

[
parking id: 5
spots count: 4

{ lat: 55.93747; lon: 37.505898; complexity: NEWBIE }
{ lat: 55.93747; lon: 37.50592; complexity: NEWBIE }
{ lat: 55.93747; lon: 37.505969; complexity: NEWBIE }
{ lat: 55.93747; lon: 37.505992; complexity: NEWBIE }
]

[
parking id: 6
spots count: 0
]

[
parking id: 7
spots count: 1

{ lat: 55.937888; lon: 37.506473; complexity: NEWBIE }
]

[
parking id: 8
spots count: 3

{ lat: 55.937933; lon: 37.506439; complexity: NEWBIE }
{ lat: 55.937933; lon: 37.506471; complexity: NEWBIE }
{ lat: 55.937936; lon: 37.506585; complexity: NEWBIE }
]

[
parking id: 9
spots count: 0
]

[
parking id: 10
spots count: 7

{ lat: 55.93768; lon: 37.505971; complexity: NEWBIE }
{ lat: 55.937665; lon: 37.50597; complexity: NEWBIE }
{ lat: 55.937649; lon: 37.505969; complexity: NEWBIE }
{ lat: 55.937556; lon: 37.505966; complexity: NEWBIE }
{ lat: 55.937541; lon: 37.505965; complexity: NEWBIE }
{ lat: 55.937525; lon: 37.505964; complexity: NEWBIE }
{ lat: 55.937461; lon: 37.505962; complexity: NEWBIE }
]

[
parking id: 11
spots count: 6

{ lat: 55.937892; lon: 37.506102; complexity: NEWBIE }
{ lat: 55.937863; lon: 37.506101; complexity: NEWBIE }
{ lat: 55.937834; lon: 37.506099; complexity: NEWBIE }
{ lat: 55.937805; lon: 37.506097; complexity: NEWBIE }
{ lat: 55.937776; lon: 37.506095; complexity: NEWBIE }
{ lat: 55.93769; lon: 37.50609; complexity: EXPERIENCED }
]

[
parking id: 12
spots count: 7

{ lat: 55.937663; lon: 37.506088; complexity: NEWBIE }
{ lat: 55.937596; lon: 37.506084; complexity: NEWBIE }
{ lat: 55.937579; lon: 37.506083; complexity: NEWBIE }
{ lat: 55.937563; lon: 37.506082; complexity: NEWBIE }
{ lat: 55.937546; lon: 37.506081; complexity: NEWBIE }
{ lat: 55.937529; lon: 37.506079; complexity: NEWBIE }
{ lat: 55.937446; lon: 37.506074; complexity: NEWBIE }
]

[
parking id: 13
spots count: 4

{ lat: 55.93794; lon: 37.506256; complexity: NEWBIE }
{ lat: 55.937845; lon: 37.506253; complexity: NEWBIE }
{ lat: 55.93782; lon: 37.506252; complexity: NEWBIE }
{ lat: 55.937795; lon: 37.506252; complexity: NEWBIE }
]

[
parking id: 14
spots count: 5

{ lat: 55.93767; lon: 37.506248; complexity: NEWBIE }
{ lat: 55.937619; lon: 37.506247; complexity: NEWBIE }
{ lat: 55.937601; lon: 37.506246; complexity: NEWBIE }
{ lat: 55.937583; lon: 37.506246; complexity: NEWBIE }
{ lat: 55.937564; lon: 37.506245; complexity: NEWBIE }
]

[
parking id: 15
spots count: 1

{ lat: 55.937768; lon: 37.506386; complexity: NEWBIE }
]

[
parking id: 16
spots count: 6

{ lat: 55.937518; lon: 37.506276; complexity: NEWBIE }
{ lat: 55.937505; lon: 37.506288; complexity: NEWBIE }
{ lat: 55.937493; lon: 37.5063; complexity: NEWBIE }
{ lat: 55.93748; lon: 37.506312; complexity: NEWBIE }
{ lat: 55.937468; lon: 37.506324; complexity: NEWBIE }
{ lat: 55.937455; lon: 37.506336; complexity: NEWBIE }
]

[
parking id: 17
spots count: 5

{ lat: 55.937368; lon: 37.506432; complexity: NEWBIE }
{ lat: 55.937353; lon: 37.506441; complexity: NEWBIE }
{ lat: 55.937304; lon: 37.506472; complexity: NEWBIE }
{ lat: 55.93729; lon: 37.506481; complexity: NEWBIE }
{ lat: 55.93725; lon: 37.506505; complexity: NEWBIE }
]

[
parking id: 18
spots count: 1

{ lat: 55.937464; lon: 37.506453; complexity: NEWBIE }
]

[
parking id: 19
spots count: 0
]

